queue_services/entity-emailer/src/entity_emailer/email_processors/amalgamation_notification.py

```diff
@@ -134,30 +135,60 @@ def _get_pdfs(
 def process(email_info: dict, token: str) -> dict:  # pylint: disable=too-many-locals, , too-many-branches
     """Build the email for Amalgamation notification."""
     logger.debug('filing_notification: %s', email_info)
+    amalgamation_application_names = {
```

Comment:
We could use legal_api/core/meta/filing.py to get displayName but it will cause issues when we merge legal name branch (as we won't keep legal-api dependency anymore)